java/test/jmri/TestArchitectureTest.java

```diff
@@ -165,4 +167,28 @@ static public void tearDown() {
         .should()
         .haveModifier(JavaModifier.ABSTRACT);
 
+    @ArchTest
```

Comment:
I recommend that you add a short JavaDoc to explain what this rule does, like the other rules have.